src/java/com/zimbra/qa/selenium/projects/ajax/tests/calendar/meetings/organizer/singleday/viewinvite/ViewInviteWithDisplayMailPreference.java

```diff
@@ -51,14 +51,21 @@ public ViewInviteWithDisplayMailPreference() throws HarnessException {
 	)
 	public void ViewHTMLInviteWithHTMLPreference_01() throws HarnessException {
 		
+		String fontfamilyData = "";
+		if (ConfigProperties.getStringProperty("browser").contains("firefox")) {
+			fontfamilyData = "<div><span style=\"font-family: &quot;comic sans ms&quot;,&quot;comic sans&quot;,sans-serif; font-size: 14pt;\">Number list below :</span>&nbsp;</div>";
+		} else {
+			fontfamilyData = "<div><span style=\"font-family: &quot;comic sans ms&quot;, &quot;comic sans&quot;, sans-serif; font-size: 14pt;\">Number list below :</span> </div>";
+		}
+		
 		// Import appointment using text mime
 		final String subject = "multiline HTML body";
 		final String boldContent = "<div><strong>BoldString</strong></div>";
 		final String italicContent = "<div><em>ItalicString</em></div>";
 		final String underlineContent ="<div><span style=\"text-decoration: underline;\">Underline String</span></div>";
 		final String colorFontContent ="<div><span style=\"color: rgb(255, 0, 0);\">Red Color text</span></div>";
 		final String colorBackgroundContent ="<div><span style=\"background-color: rgb(0, 128, 0);\">Green Background</span></div>";
-		final String fontFamilySizeContent ="<br /><div><span style=\"font-family: &quot;comic sans ms&quot;, &quot;comic sans&quot;, sans-serif; font-size: 14pt;\">Number list below :</span>�</div><br />";
+		final String fontFamilySizeContent = fontfamilyData;

```

Comment:
Remove one of the variable, both are doing same. fontFamilySizeContent = fontfamilyData;. If you use fontfamilyData name then please use camel case variable name for e.g. fontFamilyData.